bcda/models/fhir/alr/v1/coverage.go

```diff
@@ -1,14 +1,17 @@
-package alr
+package v1
 
 import (
+	"time"
+
+	"github.com/CMSgov/bcda-app/bcda/models/fhir/alr/utils"
 	fhirdatatypes "github.com/google/fhir/go/proto/google/fhir/proto/stu3/datatypes_go_proto"
 	fhirmodels "github.com/google/fhir/go/proto/google/fhir/proto/stu3/resources_go_proto"
 )
 
 // This part of the package houses the logical to create coverage resource type data
 
 // coverage takes a beneficiary and their respective K:V enrollment and returns FHIR
-func coverage(mbi string, keyValue []kvPair) *fhirmodels.Coverage {
+func coverage(mbi string, keyValue []utils.KvPair, lastUpdated time.Time) *fhirmodels.Coverage {
```

Comment:
Do we want to add in any error handling here (and the other resources)? Thinking especially for resources with non-mandatory columns - if we fail to populate certain fields, will that read as the value not being present in the original file? (Maybe not an issue here where everything is a string, but for resources that rely on regex matches, e.g., how do we handle a k:v that has a non-nil value that doesn't match any patterns)?

Reply:
I think for now, we don't include any K:v not mapped to a pattern. But logging it would be a great idea.